modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro

```diff
@@ -204,6 +204,20 @@ definition {
 			var storageDLFolderPath = "/${storageDLFolderPath}";
 		}
 
+		if (!(isSet(isReadOnly))) {
+			var isReadOnly = "false";
+		}
+
+		if (isSet(readOnlyExpression)) {
+			var single_quote = "'";
+			var scape_quotes = "'\''";
+
+			var readOnlyExpression = StringUtil.replace(${readOnlyExpression}, ${single_quote}, ${scape_quotes});
```

Comment:
I tried to implement without using the `single_quote` and `scape_quotes`, like this:

- var readOnlyExpression = StringUtil.replace(${readOnlyExpression}, "`'`", "`'\''`");

But according to the errors shown at terminal, it's treating as many strings arguments, not only the three necessary. The solution I came up was using variables.